build.xml

```diff
@@ -416,7 +416,7 @@
             <exclusion groupId="hsqldb" artifactId="hsqldb"/> <!-- outdated version -->
             <exclusion groupId="antlr" artifactId="antlr"/> <!-- outdated version -->
           </dependency>
-          <dependency groupId="net.java.dev.jna" artifactId="jna" version="4.0.0"/>
+          <dependency groupId="net.java.dev.jna" artifactId="jna" version="5.8.0"/>
```

Comment:
Going through this [ticket]([CASSANDRA-17019](https://issues.apache.org/jira/browse/CASSANDRA-17019)), do we need to bump snappy as well?